### ISSUE TITLE
Printing e-mails

### ISSUE THREAD

**Issue by maxmeyer** [gauteh/astroid#71]
Printing e-mails with `sup` was always a hassle. Have you any plans to render mails with HTML/CSS and make them easy to print directly from astroid? Or is this already supported?

**Comment by gauteh** [comment]
Dennis Günnewig writes on January 19, 2016 22:29:
> Printing e-mails with `sup` was always a hassle. Have you any plans to render mails with HTML/CSS and make them easy to print directly from astroid? Or is this already supported?
 
I haven't really thought too much about it, the CSS file is largely
based on Geary which seems to have print support. There is also a print
stylesheet, so it should not be too hard to add. Perhaps it is possible
to hook into webkit here.

**Comment by gauteh** [comment]
https://git.gnome.org/browse/geary/tree/src/client/conversation-viewer/conversation-viewer.vala#n1790

```
    private void on_print_message(Geary.Email message) {
        try {
            email_to_element.get(message.id).get_class_list().add("print");
            web_view.get_main_frame().print();
            email_to_element.get(message.id).get_class_list().remove("print");
        } catch (GLib.Error error) {
            debug("Hiding elements for printing failed: %s", error.message);
        }
    }

```

Should definetely not be too hard, needs some testing with the UI
changes we have done though!

**Comment by gauteh** [comment]
Please test #72.

**Comment by gauteh** [comment]
Fixed in #72.

**Comment by maxmeyer** [comment]
Wow, great! Thanks a lot.
